lib/tasks/publishing_api.rake

```diff
@@ -188,6 +188,18 @@ namespace :publishing_api do
     puts "Finished queuing items for Publishing API"
   end
 
+  desc "Republish all documents with official attachments"
+  task republish_documents_with_official_attachments: :environment do
+    attachment_ids = Attachment.where("unnumbered_command_paper = true OR command_paper_number != '' OR unnumbered_hoc_paper = true OR hoc_paper_number != ''").pluck(:attachable_id).uniq
```

Comment:
The internet suggests it might be possible to write this as:
```ruby
Attachment
  .where(unnumbered_command_paper: true)
  .or("command_paper_number != ''")
  .or(unnumbered_hoc_paper: true)
  .or("hoc_paper_number != ''")
  .pluck(:attachable_id)
  .uniq
```

Reply:
Just tried this in integration and it doesn't like giving a string to `or` like this. I can mess around with something like `Attachment.where(unnumbered_command_paper: true).or(Attachment.where("command_paper_number != ''")).......` but I don't think that's better

Reply:
Ok, no problem. If this was more longer lived code, maybe it would be worth putting up more of a fight against ActiveRecord, but seen as it's a temporary Rake task, I think it's wise to leave it be now.